src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -161,6 +162,7 @@ def load_arguments(self, _):
         c.argument('snapshot_id', type=str, validator=validate_snapshot_id)
         c.argument('enable_oidc_issuer', action='store_true', is_preview=True)
         c.argument('crg_id', validator=validate_crg_id, is_preview=True)
+        c.argument('message_of_the_day', type=str)
```

Comment:
Better to leave a note here, explain that the windows nodepool cannot be created (at this time) so verification is not needed.